src/modules/auth/index.js

```diff
@@ -122,4 +135,67 @@ export default class AuthModule extends Module {
 		return true;
 	}
 
+
+	async _isUserAuthorized() {
+		let resp = false;
```

Comment:
Well, I'm not sure if you actually do authorization check here.
I see that you download tenants.

What you actually need to do, it to call 'rbac' API endpoint and decide on the result.

Tenants should be downloaded else where - likely not in auth. module (that should to some extent work on non-tenant installation as well). It was also reason for my previous comment, that "assigned" tenants should be part of the "userinfo" call, handled elsewhere.

Reply:
@ateska Well, I understand and correct me if I am wrong, but there is a thing, that from `userinfo` the app will get only the tenants assigned to a user and not all the tenants of the application. So then the `rbac` will always pass and there is nothing to check. Because `rbac` should validate, if the users tenants are authorized for the application, am I right? So I need to check all the application tenants (not the users tenants) and validate them with rbac based on user (the information about which user is taken from `acces token`). So that is why authorization needs to get all the tenants of the application - and this is what (now) you dont get from `userinfo`.

In the section, you have commented - I actually download application tenants and then checking them with `rbac` call. And there I validate, if the user tenants are valid for the application and there I get the result if it is valid and check the authorization. The one of the possibilities is that I move the downloading of applicaiton tenants to the Application.js file and then I will pass it to Auth module to avoid loading tenants in the Auth module.